src/Tests.IntegrationTests/appsettings.json

```diff
@@ -1,5 +1,6 @@
 {
   "Bot1Url": "http://localhost:9012",
+  "Bot1ApiKey":  "testing",
```

Comment:
Just a note to say that this will need to be updated in the CI pipelines too I think (separate PR).